docs/install/uninstallEverest.md

```diff
@@ -8,98 +8,227 @@ You can run the commands below to remove all Everest resources including:
 !!! note alert alert-primary "Warning"
     Uninstalling Everest will remove all database clusters and associated data from the Kubernetes cluster, including backups!
 
-    Before proceeding with Custom Resource Definitions (CRD) cleanup, check that any custom resources linked with CRDs are removed. This is a crucial step because deleting CRDs can potentially cause issues with any custom resources that depend on those definitions within the cluster.
 
-To uninstall Everest:
+To uninstall Percona Everest:
 {.power-number}
 
-1. Identify the namespace:
 
-    ```sh
-    export EVEREST_NS=percona-everest
-    ```
-
-2. Uninstall Everest:
+1. Uninstall Everest:
 
     ```sh
     everestctl uninstall
     ```
 
-3. Remove ALL created database clusters:
-
-    ```sh
-    kubectl delete db --all -n percona-everest
+2. Remove the unused Custom Resource Definitions (CRDs)(https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/).
+
+    !!! caution alert alert-warning "warning"
+
+        - Ensure that you do not remove any CRDs that are being used by the operators. Removing these CRDs can lead to unexpected behavior and errors in Percona Everest.
+        - Before proceeding with Custom Resource Definitions (CRD) cleanup, check that any custom resources linked with CRDs are removed. This is a crucial step because deleting CRDs can potentially cause issues with any custom resources that depend on those definitions within the cluster.
+
+## How to remove CRDs
+
+During the installation of Everest, the following operators will be installed:
```

Comment:
Since the user is looking at the uninstall documentation maybe we should change the verb tense
```suggestion
During the installation of Everest, the following operators are installed:
```